pkg/common/bundleutil/jwks.go

```diff
@@ -0,0 +1,113 @@
+package bundleutil
+
+import (
+	"crypto/x509"
+	"encoding/json"
+
+	"github.com/spiffe/spire/pkg/common/idutil"
+	"github.com/spiffe/spire/proto/common"
+	"github.com/zeebo/errs"
+	jose "gopkg.in/square/go-jose.v2"
+)
+
+const (
+	jwtUse  = "spiffe-jwt"
+	x509Use = "spiffe-x509"
+)
+
+func JWTJWKSBytesFromBundle(bundle *Bundle) ([]byte, error) {
+	jwksBytes, err := json.Marshal(JWTJWKSFromBundle(bundle))
+	if err != nil {
+		return nil, errs.Wrap(err)
+	}
+	return jwksBytes, nil
+}
+
+func JWTJWKSFromBundle(bundle *Bundle) *jose.JSONWebKeySet {
+	jwks := new(jose.JSONWebKeySet)
+	for keyID, jwtSigningKey := range bundle.JWTSigningKeys() {
+		jwks.Keys = append(jwks.Keys, jose.JSONWebKey{
+			Key:   jwtSigningKey,
+			KeyID: keyID,
+			// TODO: fill in with proper use value when it is known
```

Comment:
We may create an issue for this so we have more visibility that this is missing?

Reply:
That's a great idea. I will file one.